packages/mdc-textfield/mdc-textfield.scss

```diff
@@ -510,18 +516,18 @@
   height: initial;
   transition: none;
   border: 1px solid $mdc-textarea-border-on-light;
-  border-radius: 4px;
 
   @include mdc-theme-dark(".mdc-textfield") {
     border-color: $mdc-textarea-border-on-dark;
   }
 
   // stylelint-disable plugin/selector-bem-pattern
   .mdc-textfield__input {
+    @include mdc-text-field-corner-radius-all(2px);
```

Comment:
I THIIIINK you need to update your `mdc-text-field-corner-radius` mixin to be like this

```
@mixin mdc-text-field-corner-radius($position, $radius) {
  .mdc-textfield--box, .mdc-textfield--textarea {
    border-#{$position}-radius: $radius;
  }
  .mdc-textfield--textarea .mdc-textfield__input {
    border-#{$position}-radius: $radius - 2;
  }
}
```

Or something like that. The mixin should definitely handle keeping `.mdc-textfield--textarea` and `.mdc-textfield__input` in sync.

Reply:
Hm. Then my other idea (of only needing one mixin that does what `-all` does) might fly out the window.